docs/introduction/installation.mdx

```diff
@@ -7,130 +7,115 @@ description: Pester is a cross-platform PowerShell module for testing your Power
 
 ## Compatibility
 
-Pester is compatible with every version of Windows that can run at least PowerShell 3.
-That includes: *Windows 10, 8, 7, Vista*, and their respective Server versions *2016, 2012 R2, 2012, 2008 R2* and *2008*.
+Pester is a cross-platform module that runs on Windows, Linux, MacOS and anywhere else supported by PowerShell. It is tested and compatible with:
+- Windows PowerShell 3.0 - 5.1
+- PowerShell 6.0.4 and above
 
-## Installing from PSGallery on Windows 10 or Windows Server 2016
+## Installing from PSGallery
 
-Windows 10 and Windows Server 2016 make installing and updating PowerShell modules extremely simple by providing `Install-Module` and `Update-Module` cmdlets.
-Unfortunately there are some complications specific to Pester that we cannot avoid.
+### Linux & MacOS
 
-Pester version 3.4.0 ships as part of Windows 10 and Windows server 2016, and that distribution conflicts with the standard module update mechanism.
-It is not possible to update the built-in Pester to a newer version, using the `Update-Module` cmdlet.
-
-Instead you need to perform a new side-by-side installation of Pester, because `Install-Module` detects the current installation of Pester is signed with a different signature than the one you are installing.
-
-Here's the command you need to run _as administrator_ in order to get the latest version of Pester:
+From a PowerShell session run:
 
 ```powershell
-Install-Module -Name Pester -Force -SkipPublisherCheck
-```
-
-The `-SkipPublisherCheck` option is typically required for machines that only have v3.4.0 installed as mentioned above.
-That particular version of the module was signed by Microsoft to be shipped in-box, which is not the case for later versions which are community maintained and signed with a different certificate.
-Attempting to install the module without that option may fail.
-
-If you receive the following warning when trying to install the module, you may need to explicitly enable TLS 1.2:
+# For all users
+Install-Module -Name Pester
```

Comment:
CurentUser scope is (largely) the default nowadays. Maybe change this to say # Install to default location ? and then optionally show how to install for all users or current user, or maybe just link to official doc, we don't need to repeat that here.

from [doc](https://learn.microsoft.com/en-us/powershell/module/powershellget/install-module?view=powershellget-3.x): When no Scope is defined, the default is set based on the PowerShellGet version.

In PowerShellGet 1.x versions, the default is AllUsers, which requires elevation for install.
For PowerShellGet versions 2.0.0 and above in PowerShell 6 or higher:
The default is CurrentUser, which doesn't require elevation for install.
If you are running in an elevated session, the default is AllUsers.

Reply:
Thanks for the quick review! 🙂
Better in the latest commit?